### ISSUE TITLE
openmp_supported not defined

### ISSUE THREAD

**Issue by cmey** [IntelLabs/ParallelAccelerator.jl#152]
On released, `master` and `devel0.6` branches of `ParallelAccelerator.jl`:

```
using ParallelAccelerator

...

ERROR: LoadError: LoadError: UndefVarError: openmp_supported not defined
Stacktrace:
 [1] include_from_node1(::String) at ./loading.jl:569
 [2] include(::String) at /Applications/Julia-0.6.app/Contents/Resources/julia/lib/julia/sys.dylib:?
 [3] include_from_node1(::String) at /Applications/Julia-0.6.app/Contents/Resources/julia/lib/julia/sys.dylib:?
 [4] eval(::Module, ::Any) at /Applications/Julia-0.6.app/Contents/Resources/julia/lib/julia/sys.dylib:?
 [5] _require(::Symbol) at /Applications/Julia-0.6.app/Contents/Resources/julia/lib/julia/sys.dylib:?
 [6] require(::Symbol) at /Applications/Julia-0.6.app/Contents/Resources/julia/lib/julia/sys.dylib:?
while loading /Users/cmey/.julia/v0.6/ParallelAccelerator/src/cgen.jl, in expression starting on line 156
while loading /Users/cmey/.julia/v0.6/ParallelAccelerator/src/ParallelAccelerator.jl, in expression starting on line 260
```

```
julia> versioninfo()
Julia Version 0.6.0
Commit 903644385b (2017-06-19 13:05 UTC)
Platform Info:
  OS: macOS (x86_64-apple-darwin13.4.0)
  CPU: Intel(R) Core(TM) i7-4980HQ CPU @ 2.80GHz
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Haswell)
  LAPACK: libopenblas64_
  LIBM: libopenlibm
  LLVM: libLLVM-3.9.1 (ORCJIT, haswell)
```

**Comment by DrTodd13** [comment]
Still happening in the released version of PA for 0.6?

**Comment by cmey** [comment]
Nope, works like a charm!
